mailbox/api/src/test/java/org/apache/james/mailbox/events/EventDeadLettersContract.java

```diff
@@ -460,7 +460,17 @@ default void groupsWithFailedEventsShouldReturnEmptyWhenNoStored() {
         }
     }
 
-    interface EventDeadLettersHealthCheckContract extends EventDeadLettersContract {
+    interface HasEventContract extends EventDeadLettersContract {
+        @Test
+        default void hadEventShouldReturnFalseOnNothingStored() {
```

Comment:
what's the point of empty tests?

Reply:
It is a right case.Why should i dont tested it

Reply:
at this point, the method is just empty

Reply:
I just find out what you mean.